package.json

```diff
@@ -52,9 +52,9 @@
     "vscode-test-utils": "^0.0.8"
   },
   "scripts": {
-    "precompile": "node ./node_modules/vscode/bin/install",
-    "compile": "tsc",
-    "lint": "tslint --project tsconfig.json",
+    "postinstall": "node ./node_modules/vscode/bin/install",
```

Comment:
Come to think of it the only time `vscode-test` needs to be redownloaded is when there is a new version of `vscode` released. Considering though that `vscode-test` is needed only when on local or CI testing, I think it's safe.